src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -19,24 +20,25 @@ CaptureViewElement::CaptureViewElement(CaptureViewElement* parent, const Viewpor
                                        const TimeGraphLayout* layout)
     : viewport_(viewport), layout_(layout), parent_(parent) {
   ORBIT_CHECK(layout != nullptr);
+
+  static uint32_t uid_counter = 0;
```

Comment:
What about making this an atomic?

```suggestion
  static std::atomic<uint32_t> uid_counter = 0;
```

Reply:
Yes, I was thinking the same, that should be an easy way to be safe. Thanks!